Shared/Supporting Files/Views/WebView.swift

```diff
@@ -41,19 +41,18 @@ struct WebView: UIViewRepresentable {
     }
     
     class Coordinator: NSObject, WKNavigationDelegate {
-        func webView(
-            _ webView: WKWebView,
-            decidePolicyFor navigationAction: WKNavigationAction,
-            decisionHandler: @escaping (WKNavigationActionPolicy) -> Void
-        ) {
+        func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction) async -> WKNavigationActionPolicy {
             switch navigationAction.navigationType {
             case .linkActivated:
-                if let url = navigationAction.request.url, UIApplication.shared.canOpenURL(url) {
-                    UIApplication.shared.open(url)
+                if let url = navigationAction.request.url,
+                   await UIApplication.shared.canOpenURL(url) {
+                    Task { @MainActor in
```

Comment:
Use deteched task 
```suggestion
                    Task.detached { @MainActor in
```

Reply:
I'm not sure there is a difference here?